confidential-data-hub/hub/protos/api.proto

```diff
@@ -18,10 +18,22 @@ message GetResourceResponse {
     bytes Resource = 1;
 }
 
+message SecureMountRequest {
+    bytes mountInfo = 1;
+}
+
+message SecureMountResponse {
+    bytes mountPath = 1;
```

Comment:
only a little question: as path is a string, is it better to use a `string` rather than a `bytes`?